src/Vlingo.Common/BasicCompletes.cs

```diff
@@ -97,7 +97,7 @@ public virtual ICompletes<T> AndThenConsume(T failedOutcomeValue, System.Action<
 
         public virtual ICompletes<T> AndThenConsume(System.Action<T> consumer) => AndThenConsume(-1L, default(T), consumer);
 
-        public virtual O AndThenInto<F, O>(long timeout, F failedOutcomeValue, Func<T, O> function)
```

Comment:
In .NET Api it is more idiomatic to use TimeSpan or int32 but I guess it was to comply to Java version